satpy/readers/mirs.py

```diff
@@ -356,6 +356,17 @@ def _fill_data(self, data_arr, attrs):
             data_arr = data_arr.where(data_arr != fill_value, fill_out)
         return data_arr, attrs
 
+    def _get_valid_range(self, data_arr, attrs):
```

Comment:
```suggestion
    def _apply_valid_range(self, data_arr, attrs):
```

Really nitpicking here. Not necessary unless you make other changes.